lib/perso.ml

```diff
@@ -4946,6 +5043,23 @@ let print_foreach conf base print_ast eval_expr =
           List.iter (print_ast env ep) al)
         (get_surnames_aliases p)
   in
+  (* carrousel *)
+  let print_foreach_img_in_keydir env al ((p, p_auth) as ep) old =
+    let list = Image.get_keydir_files conf base p old in
+    if (not p_auth) && is_hide_names conf p then ()
```

Comment:
It should not be needed to check permission here, It should be done in the Image module

Reply:
I mimicked the print_foreach_surname_alias code above. 
p_auth checks are everywhere in perso.ml!!

Reply:
One of the main goal of making a Image module, was to make sure we correctly check permission when showing portrait, instead of doing it manually everywhere.
Here are you sure the permission is correct with just `(not p_auth) && is_hide_names conf p` and not something more complexe like we have in `Image.has_access_to_portrait`?

Reply:
I have added `no_image` in the test (`no_image_for_visitor=yes`)

Reply:
`Image.has_access_to_portrait` might be too restrictive for visitors when `no_image_for_visitor` is not set to yes